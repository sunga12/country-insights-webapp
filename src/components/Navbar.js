import { Link } from 'react-router-dom';
import mic from '../assets/microphone-solid.svg';
import settings from '../assets/gear-solid.svg';
import back from '../assets/chevron-left-solid.svg';

function Navbar() {
  return (
    <>
      <nav>
        <Link to="/">
          <img className="icon" src={back} alt="back" />
        </Link>
        <p>
          Countries
        </p>
        <img className="icon" src={mic} alt="audio" />
        <img className="icon" src={settings} alt="settings" />
      </nav>
    </>
  );
}

export default Navbar;
