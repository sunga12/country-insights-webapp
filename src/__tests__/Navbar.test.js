import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import { Provider } from 'react-redux';
import Store from '../redux/store';
import Navbar from '../components/Navbar';

it('Navbar component test', () => {
  const tree = renderer.create(
    <MemoryRouter>
    <Provider store={Store}>
      <Navbar />
    </Provider>
  </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
