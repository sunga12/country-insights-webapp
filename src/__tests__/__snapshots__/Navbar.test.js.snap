// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component test 1`] = `
<nav>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="back"
      className="icon"
      src="chevron-left-solid.svg"
    />
  </a>
  <p>
    Countries
  </p>
  <img
    alt="audio"
    className="icon"
    src="microphone-solid.svg"
  />
  <img
    alt="settings"
    className="icon"
    src="gear-solid.svg"
  />
</nav>
`;
